src/devices/airzone.ts

```diff
@@ -0,0 +1,53 @@
+import {Definition} from '../lib/types';
```

Comment:
Please also add this file to https://github.com/AlexisPolegato/zigbee-herdsman-converters/blob/master/src/devices/index.ts

Reply:
@AlexisPolegato can you check this one?

Reply:
@Koenkk  I've added here : [3a299a1a0c11186122de53242c1243b92d4b72d7](https://github.com/Koenkk/zigbee-herdsman-converters/pull/7405/commits/3a299a1a0c11186122de53242c1243b92d4b72d7)